waspc/src/Wasp/Generator/SdkGenerator.hs

```diff
@@ -359,12 +362,18 @@ genDbClient :: AppSpec -> Generator FileDraft
 genDbClient spec = do
   areThereAnyEntitiesDefined <- not . null <$> getEntitiesForPrismaSchema spec
 
-  let tmplData = object ["areThereAnyEntitiesDefined" .= areThereAnyEntitiesDefined]
+  let tmplData =
+        object
+          [ "areThereAnyEntitiesDefined" .= areThereAnyEntitiesDefined,
+            "prismaSetupFn" .= GJI.jsImportToImportJson (extImportToJsImport <$> maybePrismaSetupFn)
```

Comment:
Franjo recently extracted these two calls into its own function. So keep that in mind after you merge main.